docs/UserGuide.md

```diff
@@ -13,151 +27,211 @@ PeopleSoft is a desktop app for **calculating the salary for shift-based contrac
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your Computer. Follow [this guide](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html#GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A) to do so.
 
-1. Download the latest `peoplesoft.jar` from [here]().
+2. Download the latest application from [here](https://github.com/AY2122S2-CS2103T-T11-4/tp/releases). Select the `peoplesoft.jar` file.
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Move it to a folder that you want it to save its data in.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br><br>
```

Comment:
"similar to the below" feels a little awkwardly expressed, maybe "as pictured" or something